Examples/generate_icons.py

```diff
@@ -254,7 +254,7 @@ def getGraphicsForClass(modelicaClass):
             graphicsObj['fillPattern'] = g[11]
             graphicsObj['lineThickness'] = float(g[12])
             graphicsObj['extent'] = [[float(g[13]), float(g[14])], [float(g[15]), float(g[16])]]
-            graphicsObj['textString'] = g[17].strip('"').encode('utf-8')
```

Comment:
Seems to work fine without encoding, the `decode()` there was wrong.
Why not remova also `encode()` in line 300?

Reply:
I overlooked that one. But there are two more and I'm not 100% sure of the impact. So perhaps somebody else can have a look at those.